src/bdd_isaacsim_exec/behave.py

```diff
@@ -37,7 +38,7 @@
 PANDA_MAX_EE_SPEED_STD = 0.22451
 PANDA_SPEED_THRESHOLD = PANDA_MAX_EE_SPEED_MEAN + 2 * PANDA_MAX_EE_SPEED_STD
 SPEED_THRESHOLD = 1.1
-
+USE_LIVESTREAM = os.getenv("ISAAC_LIVESTREAM", "0") == "1"
```

Comment:
Why don't you use `**kwargs` to pass this flag and the other stream configs? Then you can propagate settings in `environment.py`